examples/inverse/plot_morph_VolSourceEstimate.py

```diff
@@ -0,0 +1,282 @@
+nonlinear morph, estimated based on respective transformation from the
+subject's anatomical T1 (brain) to fsaverage T1 (brain). Afterwards the
+transformation will be applied to the beamformer result. Affine transformations
+are computed based on the mutual information. This metric relates structural
+changes in image intensity values. Because different still brains expose high
+structural similarities this method works quite well to relate corresponding
+features [1]_. The nonlinear transformations will be performed as
+Symmetric Diffeomorphic Registration using the cross-correlation metric [2]_.
+
+.. note:: This example applies downsampling to all volumes in order to speed up
+        computation. In a real case scenario you might want to reconsider if
+        and to what extend resliceing might be necessary.
+
+References
+----------
+
+.. [1] Mattes, D., Haynor, D. R., Vesselle, H., Lewellen, T. K., &
+        Eubank, W. (2003). PET-CT image registration in the chest using
+        free-form deformations. IEEE transactions on medical imaging, 22(1),
+        120-128.
+
+.. [2] Avants, B. B., Epstein, C. L., Grossman, M., & Gee, J. C. (2009).
+        Symmetric Diffeomorphic Image Registration with Cross- Correlation:
+        Evaluating Automated Labeling of Elderly and Neurodegenerative Brain,
+        12(1), 26-41.
+
+"""
+# Author: Tommy Clausner <tommy.clausner@gmail.com>
+#
+# License: BSD (3-clause)
+
+from os import makedirs, path
+
+import matplotlib.pylab as plt
+import mne
+import nibabel as nib
+import numpy as np
+from dipy.align import imaffine, imwarp, reslice
+from mne.beamformer import make_lcmv, apply_lcmv
+from mne.datasets import sample
+from mne.externals.h5io import write_hdf5, read_hdf5
+from nilearn.image import index_img
+from nilearn.plotting import plot_anat
+
+print(__doc__)
+
+
+###############################################################################
+# from :ref:`LCMV beamformer inverse example
+# <sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py>`. Note, that
+# in order to save the data fname needs to be set
+
+def compute_lcmv_example_data(data_path, fname=None):
+    raw_fname = data_path + '/MEG/sample/sample_audvis_raw.fif'
+    event_fname = data_path + '/MEG/sample/sample_audvis_raw-eve.fif'
+    fname_fwd = data_path + '/MEG/sample/sample_audvis-meg-vol-7-fwd.fif'
+    # Get epochs
+    event_id, tmin, tmax = [1, 2], -0.2, 0.5
+
+    # Setup for reading the raw data
+    raw = mne.io.read_raw_fif(raw_fname, preload=True)
+    raw.info['bads'] = ['MEG 2443', 'EEG 053']  # 2 bad channels
+    events = mne.read_events(event_fname)
+
+    # Set up pick list: gradiometers and magnetometers, excluding bad channels
+    picks = mne.pick_types(raw.info, meg=True, eeg=False, stim=True, eog=True,
+                           exclude='bads')
+
+    # Pick the channels of interest
+    raw.pick_channels([raw.ch_names[pick] for pick in picks])
```

Comment:
can this code be simplified? do we actually need to pick channels?

Reply:
from the example... I'll see if I can remove redundant parts from the example part in general ;)